### DISCUSSION TITLE
Unusually Short Build Time with No Errors, Resulting in Automatic Promotion to Production and Function Invocation Failure

### DISCUSSION THREAD

**Discussion by cowboycodr** [vercel/community#6673] [Help]
### Summary

I'm experiencing an issue with my project hosted on Vercel. Whenever I push changes to the project's Git repository, the Vercel Git integration triggers a deployment. The build process completes in about 11 seconds, which is significantly shorter than usual.

Despite the build process reporting no errors and indicating that everything is good to go, navigating to the page results in a "Function Invocation Failed" error. This error occurs even though the deployment is automatically pushed to production due to the absence of reported errors.

As a workaround, I have to manually deploy a preview using the Vercel CLI on my local machine, and then promote it to production. This is not ideal and I would like to understand why the automatic deployments are failing.

Any insights or assistance would be greatly appreciated.

Error
![image](https://github.com/vercel/community/assets/59800761/5c5078d9-1828-4206-8705-c42f3d48d20a)

Vercel
<img width="503" alt="image" src="https://github.com/vercel/community/assets/59800761/5280e2bc-95f9-4733-912a-a7e84dbcdc13">

### Steps to Reproduce

N/A

**Comment by amyegan** [comment]
Hey @cowboycodr. Do you see any additional information in the [build log](https://vercel.com/docs/deployments/logs) or [runtime logs](https://vercel.com/docs/observability/runtime-logs)? If you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) I can take a closer look into what's causing this

**Reply by cowboycodr** [reply]
Build logs:
<img width="1340" alt="image" src="https://github.com/vercel/community/assets/59800761/d0e76a33-91e0-402c-bc90-97d482ae1273">

The runtime logs for my last deploy via the git integration have been deleted but it just said Err 500 function invocation error
